frontend/src/components/Header.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div>
+    <v-app-bar class="navbar-custom" dark clipped-right color="#2C3E50">
+      <v-toolbar-title>
+        <a href="https://codeforpoznan.pl/">
+          <v-img
+            :src="image"
+            contain
+            max-height="20"
+            aspect-ratio="1.7"
+            max-width="500"
+            min-width="350"
+            position="left"
+          ></v-img>
+        </a>
+      </v-toolbar-title>
+      <v-spacer></v-spacer>
+      <v-tabs
+        background-color="transparent"
+        right
+        class="hidden-sm-and-down"
+        mobile-break-point="768px"
+      >
+        <template v-for="item in items">
+          <v-tab align-right @click="$vuetify.goTo(item.id)" :key="item.id">{{
+            item.name
+          }}</v-tab>
+        </template>
+      </v-tabs>
+    </v-app-bar>
+    <v-list class="navbar-custom hidden-md-and-up" dark>
```

Comment:
Not sure what that piece of code does. We should probably discuss it just for orientation. Tabs were rendered before, weren't they?